manage_breast_screening/mammograms/forms/implanted_medical_device_history_form.py

```diff
@@ -125,6 +88,19 @@ def create(self, appointment, request):
 
         return implanted_medical_device_history
 
+    def full_clean(self):
+        # if a removal_year is provided then remove it if device_has_been_removed is False
```

Comment:
I wonder if `FormWithConditionalFields` should have a way of handling this use case too (i.e. declaring that a field should only be parsed in a certain case, even if it is not required to pass a value) 🤔